aio/Dockerfile

```diff
@@ -17,7 +17,7 @@
 
 # Scratch can be used as the base image because the backend is compiled to include all
 # its dependencies.
-FROM scratch
+FROM gcr.io/distroless/static
```

Comment:
Why is it better than a `scratch` image? We don't really need extra stuff that is available inside this image.

Reply:
@floreks we are aligning all k8s "system containers" with the same base image/version, to make it easier to monitor and handle CVEs across the Estate. More info: https://github.com/kubernetes/kubernetes/issues/40248
https://github.com/kubernetes/kubernetes/issues/70249

@tallclair anything else you may want to add?

Reply:
@pjbgf if there is no logic behind changing the base image other than the desire to align all images, I don't really want to change it. Considering the security I don't see an actual advantage of changing it.

As mentioned in kubernetes/kubernetes#40248:
> Dynamically loaded libraries: some images (e.g. kubeproxy, fluentd) rely on more than a statically built binary, so basing off something like scratch is non-trivial.

In our case, Dockerfile only copies statically built binary and runs it. There are no additional steps, so we actually rely only on a single binary and compiled frontend code that is served by the binary. This does not apply to our case.

Reply:
Second @floreks -- Our image footprint is already as minimal as possible. Inheriting from distroless isn't required.

Reply:
Hey @floreks @jeefy, thank you for assessing my changes. I have now removed the rebasing of the dashboard. 

Given that this was a [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-release/20190316-rebase-images-to-distroless.md) decision, is it worth creating a issue to discuss this further?

Reply:
To add a bit of context on this:
distroless/static doesn't add any executable dependencies to the image. It only adds configuration files (and ca-certs) that are required for certain go functions, even for static binaries. For example, see this issue: https://github.com/kubernetes/kubernetes/issues/69195

It's possible that these networking functions aren't required for the dashboard, but IMO there's basically no reason not to include them, and doing so might avoid some confusing bugs if those functions are called in the future.

Reply:
Thanks for the context! With that, background I'd totally accept changing the base image to `distroless/static`

Reply:
@tallclair thank you for adding the context.

@jeefy thank you for reviewing this. I have just submitted another PR to make that change.